src/Core/src/Platform/iOS/WrapperView.cs

```diff
@@ -105,12 +105,19 @@ public override void LayoutSubviews()
 			SetBorder();
 		}
 
-		public new void Dispose()
+		internal void Disconnect()
 		{
-			DisposeClip();
-			DisposeShadow();
-			DisposeBorder();
+			MaskLayer = null;
+			BackgroundMaskLayer = null;
+			ShadowLayer = null;
+			_borderView?.RemoveFromSuperview();
+		}
 
+
+		// TODO obsolete or delete this for NET9
+		public new void Dispose()
```

Comment:
Why is this new instead of a virtual+override? Is iOS SDK using it as non-virtual?

Reply:
This exists in the original code. The diff is just kind of confusing

I'm guessing it was a mistake when this was originally added and we should just delete it on the net9 branch.